src/graphql/mutation/createInsertMutationField.js

```diff
@@ -87,7 +87,14 @@ const createPayloadType = table =>
           node: output,
         }),
       },
-
+      // Add foreign key field references.
+      ...fromPairs(
+        table.getForeignKeys().map(foreignKey => {
```

Comment:
Sorry I haven't been able to get the test suite to run locally yet :/ 

FYI This is just using existing functionality from createTableType.js